### PR TITLE
Cxp 2689 olympus

### PR DESCRIPTION
**Pull request by yasskin**
## PR Checklist

First pass at updating vulnerable dependencies.
Focused on major production dependencies.
Second pass will upgrade the development dependencies.
In the "dependencies" section, npm audit now detects zero vulnerabilities.
We worked on the Storybook .mdx files too and, though it turns out the imports do not need to be updated, the files got changed by Prettier. 
```
npm audit --production
found 0 vulnerabilities
```

Storybook can be viewed [here](https://docspot.adnxs.net/projects/lucid/CXP-2689-Olympus)

- Manually tested across supported browsers

  - [x] Chrome
  - [ ] Firefox
  - [ ] Safari

- [x] Unit tests written (`common` at minimum)
- [x] PR has one of the `semver-` labels
- [ ] Two core team engineer approvals
- [ ] One core team UX approval

### CODE REVIEW

**Comment by n8thegr8** [comment]
It makes it easier to review to commit the prettier stuff separate than actual code changes since we really don't need to review those changes.

File: package.json [hunk 1]
```diff
@@ -133,13 +134,12 @@
...
 		"eslint-plugin-react": "^7.28.0",
 		"eslint-plugin-react-hooks": "^4.3.0",
-		"eslint-plugin-storybook": "^0.5.5",
+		"eslint-plugin-storybook": "^0.5.6",
 		"glob": "^7.1.7",
 		"husky": "^3.0.0",
 		"jest": "^26.6.3",
 		"less": "^3.13.1",
 		"less-loader": "^4.1.0",
-		"marksy": "^8.0.0",
```
Comment (n8thegr8) [inline package.json:142]: Was this removed on purpose?

File: package.json [hunk 2]
```diff
@@ -149,17 +149,16 @@
 		"react-docgen-typescript-loader": "^3.3.0",
 		"react-dom": "^16.6.3",
 		"sinon": "^4.5.0",
-		"storybook-chromatic": "^3.5.5",
```
Comment (n8thegr8) [inline package.json:152]: Was this removed on purpose?

**Review by n8thegr8 — APPROVED** [review]
Just a comment and a question.  It looks good otherwise.

**Review by dreidn — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (yasskin) [reply package.json:152]: Yeah. We replaced it with the "chromatic" package which is more up to date and does the same thing.

File: package.json (hunk 1 above)
Comment (yasskin) [reply package.json:142]: Yes. Nothing was using it.

**Pull request merged**